### DISCUSSION TITLE
Weather Observation Station 12

### DISCUSSION THREAD

**Discussion by azizbd909** [hafijul233/HackerRank#22] [Q&A]
Query the list of CITY names from **STATION** that do not start with vowels and do not end with vowels. Your result cannot contain duplicates.

**Input Format**

The **STATION** table is described as follows:

![1449345840-5f0a551030-Station](https://user-images.githubusercontent.com/26342593/184507993-88ccbe10-c380-40a7-9ce0-0ffe6d462295.jpg)
where LAT_N is the northern latitude and LONG_W is the western longitude.

**Comment by mah-shamim** [comment]
/*
    Enter your query here and follow these instructions:
    1. Please append a semicolon ";" at the end of the query and enter your query in a single line to avoid error.
    2. The AS keyword causes errors, so follow this convention: "Select t.Field From table1 t" instead of "select t.Field From table1 AS t"
    3. Type your code immediately after comment. Don't leave any blank line.
*/

SELECT DISTINCT CITY FROM STATION WHERE LOWER(LEFT(CITY,1)) NOT IN ('a', 'e', 'i', 'o', 'u') AND LOWER(RIGHT(CITY,1)) NOT IN ('a', 'e', 'i', 'o', 'u');
